scalameta/contrib/src/test/scala/scala/meta/contrib/ScaladocParserSuite.scala

```diff
@@ -51,6 +51,31 @@ class ScaladocParserSuite extends FunSuite {
     assert(
       parseString(
         s"""
+         // $expectedBody
```

Comment:
Id expect an Option.

`parseString: Option[Seq[DocToken]]`

Once you are using `@root` then it would probably be something like `parseString: Option[DocTree]`